src/models/FinancingPaymentMethodSpecificOutput.ts

```diff
@@ -1,16 +1,28 @@
+import type { PaymentInstructions } from './PaymentInstructions.js';
 import type { PaymentProduct3391SpecificOutput } from './PaymentProduct3391SpecificOutput.js';
 
-/** @description Object containing the specific output details for financing payment methods (Buy Now Pay Later) */
+/**
+ * @description Object containing the specific output details for financing payment methods (Buy Now Pay Later).
+ */
 export interface FinancingPaymentMethodSpecificOutput {
   /**
-   * Format: int32
    * @description Payment product identifier - please check product documentation for a full overview of possible values.
-   *     Currently supported payment methods
-   *     * 3390 - PAYONE Secured Invoice
-   *     * 3391 - PAYONE Secured Installment
-   *     * 3392 - PAYONE Secured Direct Debit
-   * @example 3390
+   * Currently supported payment methods:
+   * - `3390` - PAYONE Secured Invoice
+   * - `3391` - PAYONE Secured Installment
+   * - `3392` - PAYONE Secured Direct Debit
+   * @minimum 0
```

Comment:
Would it be better to make this a union type, since there are only discreet values allowed? This is only viable, when the allowed values can only be changed from the SKD and not from the server itself.

Also you can link to the relevant part of the documentation. Developers will appreciate this